prisma/schema.prisma

```diff
@@ -21,7 +21,8 @@ model Post {
 
 model User {
   id            String       @default(cuid()) @id
-  name          String?
+  username          String
+  password        String
```

Comment:
what's going on with the formatting here?

Also do you have plans to clean up your database structure to remove the other models and fields that aren't necessary?

Reply:
probably the username should be unique as well

Reply:
It looks like the formatting doesn't strictly matter. It also doesn't auto format

Cleaned up database structure. Will look at adding a model 'Post' in future with details on the grocery item

Made username unique 

https://github.com/auberginearms/starterApp/commit/d649e6b3c51f8c64a52649baafec8cae8c7b4992

Reply:
I think there was an unintended change to `server/usernameAlreadyExists.ts` that looks like it'd be broken

we would not be adding a `Post` model and instead would name the model based on our needs - `Post` is so named because that project was a blog and blogs have posts